content/contributing/project-information/bevy-organization.md

```diff
@@ -0,0 +1,80 @@
+{% end %}
+
+The Bevy Organization is currently broken up into the following roles:
+
+## Project Lead
+
+Project Leads have the final call on all design and code changes within Bevy. This is to ensure a coherent vision and consistent quality of code. They are responsible for representing the project publicly and interacting with other entities (companies, organizations, etc) on behalf of the project. They choose how the project is organized, which includes how responsibility is delegated. Project Leads implicitly have the power of other roles (Maintainer, Subject Matter Expert, etc).
+
+[Carter Anderson] (`@cart`) is, for now, our singular project lead. He tries to be accountable: open to new ideas and to changing his mind in the face of compelling arguments or community consensus.
+
+[Carter Anderson]: https://github.com/cart
+
+## Maintainer
+
+Maintainers have merge rights in Bevy repos. They assess the scope of pull requests and whether they fit into the Bevy project's vision. They also serve as representatives of the Bevy Project and are often the interface between the Bevy community and the Bevy Project. They assist the Project Leads in moderating the community, handling administrative tasks, defining best practices, choosing project direction, and deciding how the project is organized.
+
+Maintainers abide by the rules described in the section on [Using Github Issues](/B0rPTFF6TUyWrM1SwS7Cng) when merging pull requests.
+
+We choose new Maintainers carefully, and only after they have proven themselves in the Bevy community. Maintainers must have a proven track record of the following:
+
+1. **A strong understanding of the Bevy Project as a whole**: our vision, our development process, and our community
+2. **Solid technical skills and code contributions across most engine areas**: Maintainers must be able to evaluate the scope of pull requests, provide complete code reviews, ensure the appropriate people have signed off on a PR, and decide if changes align with our vision for Bevy. This can only be done if Maintainers are technical experts, both generically across engine subject areas, and more specifically in the Bevy codebase.
+3. **Great social skills**: Maintainers regularly deal with and resolve "community issues". They must always present a professional and friendly face. They are representatives of the project and their actions directly reflect our goals and values. Working with them should not be painful.
+4. **Thorough reviews of other peoples' PRs**: Maintainers are the last line of defense when protecting project vision and code quality. They are also often the first people new contributors interact with. They must have a history of leaving thorough and helpful code reviews.
+5. **Ethical and trustworthy behavior**: Maintainers are granted significant administrative permissions. They must be trustworthy.
```

Comment:
"Ethical" is mentioned in the "title" but not expanded on what that actually means in the context of Bevy contributions.

Could also remove it and make the whole list item only about trust, ethical behavior should probably be noted in the Code of Conduct if it isn't already, which Maintainers implicitly follow as well as the rest of the community.

Reply:
I think this is out of scope for this PR.